centml/cli/cluster.py

```diff
@@ -71,43 +108,146 @@ def get(id):
+    func = click.option(
+        "--hardware",
+        "-h",
+        prompt="Hardware",
+        type=click.Choice(list(hw_to_id_map.keys())),
+        help="Hardware instance type",
+    )(func)
+    return func
+
+
+# Define inference specific options
+def inference_options(func):
+    func = click.option("--port", "-p", prompt="Port", type=int, help="Port to expose")(func)
+    func = click.option(
+        "--env", type=InferenceEnvType(), help="Environment variables in the format KEY=VALUE", multiple=True
+    )(func)
+    func = click.option("--min_replicas", default="1", prompt="Min replicas", type=click.IntRange(1, 10))(func)
+    func = click.option("--max_replicas", default="1", prompt="Max replicas", type=click.IntRange(1, 10))(func)
+    func = click.option("--health", default="/", prompt="Health check", help="Health check endpoint")(func)
+    func = click.option("--is_private", default=False, type=bool, prompt="Is private?", help="Is private endpoint?")(
+        func
+    )
+    func = click.option("--timeout", prompt="Max concurrency", default=0, type=int)(func)
+    func = click.option("--command", type=str, required=False, default=None, help="Define a command for a container")(
+        func
     )
+    func = click.option("--command_args", multiple=True, type=str, default=None, help="List of command arguments")(func)
+    return func
 
 
-@click.command(help="Create a new deployment")
-@click.argument("type", type=click.Choice(list(depl_type_map.keys())))
-@click.option("--name", "-n", prompt="Name", help="Name of the deployment")
-@click.option("--image", "-i", prompt="Image", help="Container image")
-@click.option("--port", "-p", prompt="Port", type=int, help="Port to expose")
-@click.option(
-    "--hardware", "-h", prompt="Hardware", type=click.Choice(list(hw_to_id_map.keys())), help="Hardware instance type"
-)
-@click.option("--health", default="/", prompt="Health check", help="Health check endpoint")
-@click.option("--min_replicas", default="1", prompt="Min replicas", type=click.IntRange(1, 10))
-@click.option("--max_replicas", default="1", prompt="Max replicas", type=click.IntRange(1, 10))
-@click.option("--username", prompt=True, default="", help="Username for HTTP authentication")
-@click.option("--password", prompt=True, default="", hide_input=True, help="Password for HTTP authentication")
-@click.option("--env", "-e", required=False, type=str, multiple=True, help="Environment variables (KEY=VALUE)")
-def create(type, name, image, port, hardware, health, min_replicas, max_replicas, username, password, env):
+# Define compute specific options
+def compute_options(func):
+    func = click.option("--username", prompt="Username", type=str, help="Username")(func)
+    func = click.option("--password", prompt="password", hide_input=True, type=str, help="password")(func)
+    func = click.option("--ssh_key", prompt="SSH key", default="", type=str, help="Would you like to add an SSH key?")(
```

Comment:
Change the help message to "Add ssh key"

Reply:
fixed